### ISSUE TITLE
[Accessibility]: Code monitoring logs: wrong usage of heading elements

### ISSUE THREAD

**Issue by lguychard** [sourcegraph/sourcegraph#35158]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34194

### Problem description

- Navigate to https://sourcegraph.com/code-monitoring?visible=4
- Witness "Code Monitoring" header is an `h1` (should be `h2`)
- Witness "Monitor name", "last run" elements are `h5` (skips heading levels)

### Expected behavior

- "Code Monitor" header is an `h2`
- "Monitor name", "last run" elements should be styled as h5, but implemented using `<div>` or `<span>`, with `aria-hidden="true"` so that they do not appear for screen readers.

### Additional details

<img width="858" alt="Screenshot 2022-05-09 at 16 06 42" src="https://user-images.githubusercontent.com/1741180/167489092-fa389baf-5493-4ef3-aa98-6f9ea2c843b9.png">

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
